.vscode/extensions.json

```diff
@@ -1,3 +1,3 @@
 {
-  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode", "orta.vscode-jest"]
+  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode", "firsttris.vscode-jest-runner"]
```

Comment:
Interesting, what does this extension do differently?

Reply:
@robdmoore i did discard that change in the end, facebook's one still has better support for test explorer, the main pain was default config for auto running jest on each save which i got rid of - it was making ide slow and sluggish and i don't see much value in that anyway since CI will prevent non passing tests anyway and i added a custom npm command to simulate `pre-commit`